lib/internal/fs/promises.js

```diff
@@ -739,6 +747,68 @@ async function readFile(path, options) {
   return PromisePrototypeFinally(readFileHandle(fd, options), fd.close);
 }
 
+async function stream(handle, dest, options) {
+  checkAborted(options.signal);
+  if (!EE) {
+    EE = require('events');
+  }
+  const BUFFER_SIZE = 128 * 2 ** 10;
+  const {
+    start = 0,
```

Comment:
This is not a good default value, we should start reading from the current position instead of the start of the file by default.